db/migrate/20220817072113_add_column_email_to_employees.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnEmailToEmployees < ActiveRecord::Migration[6.1]
```

Comment:
@MadhaviBellam why you need this migration. Pavan will take care of employees, remove it from your code. pull employee related changes once pavan will merger then into dev branch